src/plugins/WebOfTrust/util/TickerDelayedBackgroundJobFactory.java

```diff
@@ -0,0 +1,70 @@
+package plugins.WebOfTrust.util;
+
+import freenet.support.Executor;
+import freenet.support.Ticker;
+
+/**
+ * Background job factory implementation for {@link TickerDelayedBackgroundJob}s.
+ *
+ * @author bertm
+ * @see TickerDelayedBackgroundJob
+ */
+public class TickerDelayedBackgroundJobFactory
+        extends BackgroundJobFactoryBase
+        implements DelayedBackgroundJobFactory {
+    /** The default delay. */
+    private final long defaultDelay;
+    /** The default ticker. */
+    private final Ticker ticker;
```

Comment:
A custom ticker can be specified in one of the newJob(), so this should be also named "defaultTicker" to be coherent with "defaultDelay".